low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2

```diff
@@ -102,7 +94,6 @@ cpu_usage_threshold=80
 # Memory Usage Threshold value in terms of usage in percentage (i.e. 0 to 100%)
 host_memory_usage_threshold=80
 
-
 [RARITANPDU]
```

Comment:
RARITANPDU not in LDR R2, remove from here

Reply:
Okay